test/kpod_inspect.bats

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
```

Comment:
This value isn't used in this file.  Maybe change it to `redis:alpine` and use it?